### PR TITLE
create collection ver for CDM

### PR DESCRIPTION
**Pull request by Tianhao-Gu**

### CODE REVIEW

File: scripts/create_custom_collection.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        return target_dir
+
+    if os.path.exists(new_dir) and os.path.islink(new_dir) and os.readlink(new_dir) == str(target_dir):
+        return None
+
+    try:
+        os.symlink(target_dir, new_dir, target_is_directory=True)
+        return None
+    except FileExistsError:
+        raise ValueError(f"{new_dir} already exists and does not link to {target_dir} as expected")
```
Comment (MrCreosote) [inline scripts/create_custom_collection.py:46]: We have code almost exactly like this in collections... at some point we should think about what code lives where and how we might want to share code

File: scripts/create_custom_collection.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                                   ncbi_source_dir / genome_id,
+                                   TARGET_EXT) for genome_id in lineage_genome_ids]
+
+        for future in concurrent.futures.as_completed(futures):
+            try:
+                skipped_dir = future.result()
+                skipped_dirs.append(skipped_dir) if skipped_dir else None
+            except Exception as e:
+                failed_results.append((future, f"Error: {e}"))
+                print(f"Error processing {future}: {e}")
```
Comment (MrCreosote) [inline scripts/create_custom_collection.py:73]: Threading directory creation seems like overkill, with the GIL and minimal IO I'm surprised it makes a difference

**Review by MrCreosote — APPROVED** [review]
LGTM

File: scripts/create_custom_collection.py (hunk 1 above)
Comment (Tianhao-Gu) [reply scripts/create_custom_collection.py:46]: yea. This is actually from collection with a slight modification

**Pull request merged**